docker/nix-build.sh

```diff
@@ -4,6 +4,13 @@ set -euo pipefail
 SPAWN_SHELL=${SPAWN_SHELL:-false}
```

Comment:
Can we please increase `max-jobs` param since now it takes approx 10 mins to download all the derivations from `nixpkgs`?

Reply:
`--max-jobs` is now `auto` (number of cpus)